app/poros/search.rb

```diff
@@ -0,0 +1,8 @@
+class Search
+  attr_reader :total_hits, :foods
+
+  def initialize(data)
+    @total_hits = data[:totalHits]
+    @foods = data[:foods].take(10).map {|food| Food.new(food)}
+  end
+end
```

Comment:
This looks good, the only thing I'd recommend is having a `foods` method that performs the logic of converting a hash into food objects. We want to take out as much logic from the initialize method as possible. So, I'd suggest doing something like: 

```ruby
class Search
  attr_reader :total_hits

  def initialize(data)
    @total_hits = data[:totalHits]
    @food_data = data[:foods].take(10)
  end

def foods 
@food_data.map {|food| Food.new(food)}
end 
end
```